RELEASE.md

```diff
@@ -33,11 +33,20 @@ oqs-provider now also enables use of QSC algorithms during TLS1.3 handshake. The
 
 ### Algorithm updates
 
-None.
+All algorithms no longer supported in the [NIST PQC competition](https://csrc.nist.gov/projects/post-quantum-cryptography) have been removed. All remaining algorithms with the exception of McEliece have been lifted to their final round 3 variants as documented in [liboqs](https://github.com/open-quantum-safe/liboqs/blob/main/RELEASE.md#release-notes). Most notably, algorithm names for Sphincs+ have been changed to the naming chosen by its authors.
```

Comment:
We kept FrodoKEM around even though it's been eliminated from NIST since it's being standardized by ISO.

Reply:
Corrected in https://github.com/open-quantum-safe/oqs-provider/pull/175/commits/774e013f2c6c737839a4d6f683cc17f9583c68ee